packages/botonic-dx/sample-config/tsconfig.eslint.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "node_modules/botonic-dx/baseline/tsconfig.eslint.json",
```

Comment:
Should this be `@botonic/dx/...` instead of `node_modules/botonic-dx/...` as it is in other configs in `sample-config` folder?

Reply:
exactly! fixed